scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -151,25 +151,50 @@ <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.n
+                                </span>
+                            </div>
+                            <div class="sd-line-input sd-line-input--is-select">
+                                <select
+                                    ng-model="desk.edit.default_content_template"
+                                    class="sd-line-input__select"
+                                    required
+                                    data-test-id="field--default-content-template"
+                                >
                                 <option value=""></option>
-                            </select>
+                                <option ng-repeat="template in templates" value="{{template._id}}" data-test-id="template {{template.template_name}}" ng-selected="desk.edit.default_content_template === template._id">
+                                    {{ template.template_name | translate}}
+                                </option>
+
+                                </select>
+                            </div>
                         </div>
                     </div>
                     <div class="form__row" ng-show="profiles.length">
-                        <div class="sd-line-input sd-line-input--is-select">
+                        <div class="sd-line-input sd-line-input--required">
                             <label class="sd-line-input__label" translate>Default Content Profile</label>
-                            <select ng-model="desk.edit.default_content_profile"
-                                    ng-options="profile._id as profile.label for profile in profiles | orderBy: 'label'" class="sd-line-input__select">
-                                <option value=""></option>
-                            </select>
+                            <div class="sd-alert sd-alert--hollow sd-alert--small">
+                                <i class="icon-info-sign icon--blue icon--2x sd-alert__icon"></i>
+                               <span translate>
+                                    Select a default content profile which is used for articles fetched from ingest. If you don't see any, create one
+                                   <a href="#/settings/content-profiles">here</a>
+                                </span>
+                            </div>
+                            <div  class="sd-line-input sd-line-input--is-select">
+                                <select ng-model="desk.edit.default_content_profile"
+                                        class="sd-line-input__select"
+                                        required>
+                                    <option value=""></option>
+                                    <option ng-repeat="profile in profiles" value="{{profile._id}}" data-test-id="profile {{profile.label}}" ng-selected="desk.edit.default_content_profile === profile._id">
```

Comment:
Add `data-test-id` on a select instead of options. Also try avoiding dynamic test IDs where possible as they are harder to find.